pkg/security/scc/byrestrictions.go

```diff
@@ -31,20 +45,20 @@ func pointValue(constraint *securityapi.SecurityContextConstraints) int {
 	// strategies in order of least restrictive to most restrictive
 	switch constraint.SELinuxContext.Type {
 	case securityapi.SELinuxStrategyRunAsAny:
-		points += 4
+		points += runAsAnyUserWeight
```

Comment:
> Either way do we need a check (nil for map, default for switch) if something isn't recognized? Probably a good idea.

@pweil- How we can react on this case? I don't think that we should fail. I can think only about logging a warning.